AsyncDisplayKit/ASCellNode.mm

```diff
@@ -120,10 +121,10 @@ - (void)setLayerBacked:(BOOL)layerBacked
   ASDisplayNodeAssert(!layerBacked, @"ASCellNode does not support layer-backing.");
 }
 
-- (void)setNeedsLayout
+- (void)__setNeedsLayout
```

Comment:
Ah yeah that's the one I meant. Not sure if that's what's being discussed, but in theory it would be easy to modify `calculatedLayoutDidChange` (and its antecedent `_completeLayoutCalculation`) to accept the previous layout as an argument, and then have ASCellNode use that to drive its layout delegate calling, where `sizeChanged <= (self.calculatedSize != oldLayout.size)`.